runtime/marshal_jsonpb_test.go

```diff
@@ -727,6 +727,37 @@ func TestJSONPbMarshalResponseBodies(t *testing.T) {
 				}
 			},
 		},
+		{
+			input: []*examplepb.RepeatedResponseBodyOut_Response{
+				&examplepb.RepeatedResponseBodyOut_Response{},
+				&examplepb.RepeatedResponseBodyOut_Response{
+					Data: "abc",
+					Type: examplepb.RepeatedResponseBodyOut_Response_A,
+				},
+			},
+			verifier: func(json string) {
+				expected := `[{},{"data":"abc","type":"A"}]`
+				if json != expected {
+					t.Errorf("json not equal (%q, %q)", json, expected)
+				}
+			},
+		},
+		{
+			emitDefaults: true,
+			input: []*examplepb.RepeatedResponseBodyOut_Response{
+				&examplepb.RepeatedResponseBodyOut_Response{},
+				&examplepb.RepeatedResponseBodyOut_Response{
+					Data: "abc",
+					Type: examplepb.RepeatedResponseBodyOut_Response_B,
+				},
+			},
+			verifier: func(json string) {
+				expected := `[{"data":"","type":"UNKNOWN"},{"data":"abc","type":"B"}]`
+				if json != expected {
+					t.Errorf("json not equal (%q, %q)", json, expected)
+				}
+			},
+		},
```

Comment:
Could we add a test case for `EmitDefaults: false` and an empty/nil slice?

Reply:
Test case 5 (line 665) and 7 (line 684) are already testing those cases.
```
{
			input: ([]*examplepb.RepeatedResponseBodyOut_Response)(nil),
			verifier: func(json string) {
				expected := `null`
				if json != expected {
					t.Errorf("json not equal (%q, %q)", json, expected)
				}
			},
		},
```
and 
```
		{
			input: []*examplepb.RepeatedResponseBodyOut_Response{},
			verifier: func(json string) {
				expected := `[]`
				if json != expected {
					t.Errorf("json not equal (%q, %q)", json, expected)
				}
			},
		},
```